.github/workflows/conda.yml

```diff
@@ -53,6 +54,9 @@ jobs:
           mamba env export | grep -E -v '^prefix:.*$'
 
       - name: Run tests
+        env:
+            DASK_COILED__TOKEN: ${{ secrets.COILED_BENCHMARK_BOT_TOKEN }}
```

Comment:
Unfortunately I don't think secrets are available from forks. Can you re-submit this PR with a branch directly from the `coiled/coiled-runtime` repo?

Reply:
This makes sense, I forgot about this. Thanks for pointing this out. Will do!